db/seeds.rb

```
@@ -5,3 +5,25 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Image.create([{ url: 'https://www.dchen.tech/assets/images/Street_Urban/18.jpg' },
```

we should throw an exception if this fails